### PR TITLE
About us

### PR DESCRIPTION
**Pull request by JRGranell**
### Motivation

To create an About Us page that contains as an MVP the list of principles of the company and a contact form (Cain-slice).

### Testing

Test visual rendering at https://www-staging.red-badger.com/314db2f/ by navigating to the About Us page.

Principles and contact form (Cain-Slice) should be visible. Buttons on hover now change background colour according to brand guidelines.

This pull introduces the button component (with test).

### Pre-merge checklist

- [ ] Documentation
- [x] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [x] Manual testing
  - [x] Windows 7 IE 11
  - [x] Windows 10 Edge
  - [x] Mac OS El Capitan Safari
  - [x] Mac OS El Capitan Chrome
  - [x] Mac OS El Capitan Firefox
  - [x] iOS 9 Safari
  - [x] Android 6 Chrome
  - [x] Listen to the site on a screen reader
  - [x] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,4 @@ pids
 .env
 .DS_Store
 assets/state.json
+.vscode
```
Comment (JoeStanton) [inline .gitignore]: 😨

File: .gitignore [hunk 2]
```diff
@@ -17,4 +17,4 @@ npm-debug.log*
 pids
 .env
 .DS_Store
-assets/state.json
+assets/state.json
```
Comment (JoeStanton) [inline .gitignore]: Hmm - according to `.editorconfig` we should have a trailing newline https://github.com/redbadger/website-honestly/blob/master/.editorconfig#L6

Not sure if VScode will pick up on this file by default?

File: site/pages/about-us/principles-slice/principle/style.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.leftCol{
+}
+.rightCol{
+	padding-left: 30px;
+	max-width: 670px;
+}
+.number {
+	padding-top: 5px;
+  composes: boldSansSerif from "../../../../css/typography/_fonts.css";
```
Comment (JoeStanton) [inline site/pages/about-us/principles-slice/principle/style.css]: slightly odd indentation for a few of these `composes:`

**Comment by JoeStanton** [comment]
This looks good. Two minor comments!

File: site/pages/about-us/principles-slice/principle/style.css (hunk 3 above)
Comment (JRGranell) [inline site/pages/about-us/principles-slice/principle/style.css]: Yeah - I don't actually see that in my IDE...suspect it's to do with tab vs space indenting.

File: site/components/button/style.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.light:focus,
+.dark:focus,
+.yellow:focus{
+  outline: none;
+}
+
+.light:hover{
+  background: #212121 !important;
```
Comment (asavin) [inline site/components/button/style.css]: Colors should be defined in `_colors.css`

File: site/pages/about-us/principles-slice/index.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+export default function Principles() {
+  return (
+    <div className={style.principles}>
+      <div className={style.container}>
+        <div className={style.header}>Our principles.</div>
+        <div className={style.thisIsWhatWeBelieveIn}>
+          This is what we believe – and what the founders say after a couple of pints
+        </div>
+        <Principle number="01" title="A team of leaders." description="We trust our people, giving them the autonomy to make the right decisions." />
```
Comment (asavin) [inline site/pages/about-us/principles-slice/index.js]: Let's improve the semantics by making it a list, that would help screen readers to interpret this better.

File: site/pages/about-us/principles-slice/style.css [hunk 6]
```diff
@@ -0,0 +1,55 @@
+@value mediumScreen, largeScreen, maxWidth from "../../../css/_sizes.css";
+@value badgerWhite, badgerRed, badgerBlack from "../../../css/_colors.css";
+
+.principles {
+	background:badgerBlack;
```
Comment (asavin) [inline site/pages/about-us/principles-slice/style.css]: something is wrong with spacing here, it's a bit inconsistent throughout the file

File: site/slices/contact-us-slice/form/index.js [hunk 7]
```diff
@@ -45,7 +51,7 @@ class Form extends Component {
             name="message"
             defaultValue={message}
             onChange={this.handleInputChange}
-          />
+            />
```
Comment (asavin) [inline site/slices/contact-us-slice/form/index.js]: Potentially wrong indentation here

File: site/slices/contact-us-slice/form/index.js [hunk 8]
```diff
@@ -60,23 +66,20 @@ class Form extends Component {
             type="text"
             value={contact}
             onChange={this.handleInputChange}
-          />
+            />
```
Comment (asavin) [inline site/slices/contact-us-slice/form/index.js]: and here

File: site/slices/contact-us-slice/form/index.js [hunk 9]
```diff
@@ -60,23 +66,20 @@ class Form extends Component {
...
-                }}
-              >
-                Try Again
-              </button>
+              <Button
+                label='Try Again'
+                className={'fatalErrorButton'}
+                onClick={this.handleSubmit}
+                background="yellow"
+                />
```
Comment (asavin) [inline site/slices/contact-us-slice/form/index.js]: and here

File: site/slices/contact-us-slice/form/index.js [hunk 10]
```diff
@@ -86,18 +89,13 @@ class Form extends Component {
...
-                  onSubmit(this.state);
-                }}
-              >
-                Submit
-              </button>
+            <Button
+              label="Submit"
+              onClick={this.handleSubmit}
+              background="yellow"
+              />
```
Comment (asavin) [inline site/slices/contact-us-slice/form/index.js]: and here

**Pull request merged**